src/context/login-context.jsx

```diff
@@ -1,24 +1,62 @@
+import axios from "axios";
 import { createContext, useContext, useState } from "react";
-import { CartContext } from "./cart-context";
-import { WishlistContext } from "./wishlist-context";
+import { useCart } from "./cart-context";
```

Comment:
here also make one index.js .

Reply:
its because of auto import, i have already index, i have to manually import through index